addon/model.js

```diff
@@ -91,7 +91,7 @@ if (CUSTOM_MODEL_CLASS) {
       // have to use it. This is safe, because we already checked that the
       // property is not `in` the instance, so it will definitely call
       // `unknownProperty` and will not re-enter.
-      return receiver.get(key);
+      return target.get(key);
```

Comment:
1. comment is now out of date
2. there's some trickiness here, the comment should explain
  i. why we're using `target` instead of `receiver`, since it's non-obvious without knowing the `Ember.get` implementation
  ii. what trade-off is (i.e. that `this` will not be the proxy in `unknownProperty` which is something we only need to be aware of within `ember-m3` but not something that will impact users)

Reply:
Fixed